### PR TITLE
documentation contribution made by howard yoo

### PR DESCRIPTION
**Pull request by howardyoo**
- added details on object model, getting-started
- fixed build error in integration/airflow inlet and outlet doc
- fixed typo in naming.md

Signed-off-by: howardyoo <howardyoo@gmail.com>

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -58,15 +58,17 @@ To view the Marquez UI and verify it's running, open [http://localhost:3000](htt
 
 Marquez is an [LF AI & DATA](https://lfaidata.foundation) incubation project to collect, aggregate, and visualize a data ecosystem’s metadata. Marquez is the reference implementation of the OpenLineage standard.
 
-In this example, we show how you can collect dataset and job metadata using Marquez. Using the [LineageAPI](https://marquezproject.github.io/marquez/openapi.html#tag/Lineage), metadata will be collected as OpenLineage events using the run ID `d46e465b-d358-4d32-83d4-df660ff614dd`. The run ID will enable the tracking of run-level metadata over time for the job `my-job`. So, let's get started!
+In this example, we show how you can collect dataset and job metadata using [Marquez](https://marquezproject.github.io/marquez/), Using the [LineageAPI](https://marquezproject.github.io/marquez/openapi.html#tag/Lineage).
```
Comment (rossturk) [inline docs/getting-started.md:61]: One small change here - I would reword:

```
In this example, we show how you can collect dataset and job metadata using [Marquez](https://marquezproject.github.io/marquez/) and the [LineageAPI](https://marquezproject.github.io/marquez/openapi.html#tag/Lineage). 
```

**Review by rossturk — CHANGES_REQUESTED** [review]
One very small change, then we can merge this 👍

File: docs/getting-started.md (hunk 1 above)
Comment (howardyoo) [reply docs/getting-started.md:61]: Cool, I think we can do that.

**Review by rossturk — APPROVED** [review]
